examples/backends/vllm/launch/agg_omni_realtime.sh

```
@@ -0,0 +1,62 @@
+print_curl_footer <<TEST
+  # /v1/realtime is a WebSocket endpoint; drive it with the realtime client
+  # (omit --input-audio to fetch a sample clip from the vLLM-Omni repo):
+  python ${SCRIPT_DIR}/realtime_omni_client.py \\
+    --url ws://localhost:${HTTP_PORT}/v1/realtime \\
+    --model "${MODEL}" \\
+    --output-dir dynamo-realtime
+TEST
+
+
+python -m dynamo.frontend &
+FRONTEND_PID=$!
```

Shellcheck (0.11.0)

[warning] FRONTEND_PID appears unused. Verify use (or export if used externally).

(SC2034)